presentation/backend/src/presentation-backend/Utils.ts

```diff
@@ -11,6 +11,21 @@ import { Element, IModelDb } from "@itwin/core-backend";
 import { DbResult, Id64String } from "@itwin/core-bentley";
 import { Diagnostics, DiagnosticsOptions, InstanceKey } from "@itwin/presentation-common";
 
+const ecPresentation = require("@itwin/presentation-common/assets/locales/en/ECPresentation.json"); // eslint-disable-line @typescript-eslint/no-var-requires
```

Comment:
@aruniverse tried using `require("@itwin/presentation-common/lib/assets/locales/en/ECPresentation.json")` but it didn't work.

`require("@itwin/presentation-common/lib/cjs/assets/locales/en/ECPresentation.json")` seems to work. Should I use it instead?

Reply:
I think thats expected. I believe you'd need to update the `build:assets` script in presentation-common to copy to `lib/assets` not `lib/cjs/assets`

Reply:
Ok. Changed to `require("@itwin/presentation-common/lib/cjs/assets/locales/en/ECPresentation.json")`